examples/microViT.py

```diff
@@ -151,11 +151,11 @@ def forward(self, x):
         x = x.flatten(2, 3).transpose(1, 2).contiguous()  # B HW C
 
         if self.hparams.classifier == Classifier.TOKEN:
-            # prepend classification token
+            # prepend classification token to the sequence
             clf_token = (
-                torch.ones(1, batch, self.hparams.dim, device=x.device) * self.clf_token
```

Comment:
cc @SeanNaren, FYI the dimensions were broken here. We preprend the token to the sequence dimension, as suggested in the ViT paper

Reply:
thanks @blefaudeux for the headsup!